tensorflow/compiler/xla/tests/local_client_aot_test_helper.cc

```diff
@@ -71,6 +71,8 @@ int main(int argc, char** argv) {
     triple_string = "aarch64-none-linux-gnu";
   } else if (target_cpu == "x64_windows") {
     triple_string = "x86_64-pc-windows-msvc19";
+  } else if (target_cpu == "ppc") {
+    triple_string = "ppc64le-ibm-linux-gnu";
```

Comment:
This codepath needs a test case.

Reply:
When I ran the tests from TF CPU with XLA on, this `local_client_aot_test_helper ` also executed but failed with `unsupported TARGET_CPU`. So,  had to add the case for Power, after which the test passed.